src/styles/blocks/product_card.scss

```diff
@@ -0,0 +1,135 @@
+.main {
+  display: flex;
+  margin: 0 auto;
+  padding: 50px 40px;
+  flex-wrap: wrap;
+  width: 1024px;
+  justify-content: center;
+}
+
+@media (min-width: 1200px) {
+  .main {
+    width: 1200px;
+    padding: 50px 128px;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+  height: 406px;
+
+  &__imac {
```

Comment:
The class name '.card__imac' seems to be too specific for a card element. Consider using a more generic name like '.card__image'. 🖼️